Code/Framework/AzToolsFramework/AzToolsFramework/Viewport/EditorContextMenu.cpp

```diff
@@ -20,25 +19,19 @@ namespace AzToolsFramework
         if (mouseInteraction.m_mouseInteraction.m_mouseButtons.Right() &&
             mouseInteraction.m_mouseEvent == ViewportInteraction::MouseEvent::Down)
         {
-            contextMenu.m_shouldOpen = true;
             contextMenu.m_clickPoint =
                 ViewportInteraction::QPointFromScreenPoint(mouseInteraction.m_mouseInteraction.m_mousePick.m_screenCoordinates);
         }
 
-        // disable shouldOpen if right clicking an moving the mouse
```

Comment:
nitpick - looks like it should be "and" not "an"

Reply:
Good spot thanks, will fix :+1:

Reply:
Looks like that was actually a deleted line 😉